### ISSUE TITLE
Make the invite flow easier.

### ISSUE THREAD

**Issue by rahulgi** [rahafoundation/raha.io#55]
Currently, the invite flow a) isn't fully implemented and b) is pretty hard to follow. This issue attempts to address point b.

Proposal:
* `/me` redirects to the `/m/<member-id>` link for your profile. Having two different profiles is confusing. Eventually we should probably just get rid of `/me`. The `/m/..` link contains the same information as `/me` but is also the root url for the invite flow and a global locator which is preferable to the logged-in-user-relative `/me`.
* Invite instructions will be moved to the root `/m/<member-id>` page. Growing the network is our number one priority, and these instructions should be the first thing you see.
* The invite page for new members will remain `/m/<id of inviting member>/invite`.
* After creating an account, a new user will be immediately directed to their new, as of yet untrusted by the network, profile at `/m/<brand new pending member id>`.
* Once a person has created an account, the inviting member can go to their profile at `/m/<pending member id>` and hit the trust button to trust the new person.

Questions and feedback on this proposal are welcome!

**Comment by markulrich** [comment]
Seems like most of this is done! Moved last point to https://github.com/rahafoundation/raha.io/issues/90 and also created https://github.com/rahafoundation/raha.io/issues/91
